sdk/python/symbolchain/core/nem/external/ed25519.py

```diff
@@ -180,9 +180,9 @@ def scalarmult_B(e):
 def encodeint(y):
```

Comment:
ugh, should we exlude external from lints? 😬

Reply:
there weren't too many changes. i exclude js one because that would require quite a bit of changes.

Reply:
y, but it did reformat of conditions